.github/workflows/draft-release.yml

```diff
@@ -0,0 +1,43 @@
+---
+name: "Draft Release"
+
+on:
+  push:
+    tags:
+      - "*"
+
+jobs:
+  draft-release-notes:
+    needs: build-asbackup-mac
+    runs-on: ubuntu-latest
+    name: Draft Release
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+    - name: Get Latest Tag
+      run: |
+        LATEST_TAG=$(git describe --tags `git rev-list --tags --max-count=1`)
```

Comment:
curious what `rev-list` is there for?

Reply:
its function here is to list the most recent commit.